animatch/LogIn/LogInView.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  LogInView.swift
+//  animatch
+//
+//  Created by Ксения Панкратова on 23.11.2024.
+
+import SwiftUI
+
+struct LogInView: View {
+    @State private var inputPassword = ""
+    @State private var inputEmail = ""
+
+    var body: some View {
+        VStack {
+            ZStack {
+                backgroundLineGradient
+
+                VStack(spacing: 0) {
+                    Logo
+                        .padding(.bottom, 66)
+                    formsContainer
+                    buttonsContainer
+                    Spacer()
+                    footer
+                }
+            }
+        }
+        .ignoresSafeArea()
```

Comment:
nit: Как альтернативу, можешь делать так, чтобы избегать лишней вложенности. Будет читабельнее и симпотичнее

```suggestion
        VStack(spacing: 0) {
            Logo
                .padding(.bottom, 66)
            formsContainer
            buttonsContainer
            Spacer()
            footer
        }
        .frame(maxWidth: .infinity)
        .background(backgroundLineGradient)
        .ignoresSafeArea()
```

PS. nit - Разработчики используют префикс «nit:» в качестве сокращения для обозначения незначительных проблем в комментариях к коду. Т.е не обязательно править